frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -300,6 +300,22 @@ export enum Resources {
   KnativeService = 'knative',
 }
 
+export enum FuncBuilderTypes {
+  s2i = 's2i',
+  pack = 'pack',
+}
+
+export enum Runtime {
+  Node = 'node',
+  NodeJS = 'nodejs',
+  TypeScript = 'typescript',
+  Quarkus = 'quarkus',
+  Go = 'go',
+  Rust = 'rust',
+  SpringBoot = 'springboot',
+  Python = 'python',
+}
```

Comment:
Serverless function is supported for these runtimes 
`enum Runtime {
  node = 'nodejs',
  nodejs = 'nodejs',
  typescript = 'nodejs',
  quarkus = 'java',
}
`
we do not have support for these runtimes ` ['go', 'rust', 'springboot', 'python']` 
If I add git URL https://github.com/vikram-raj/hello-func-rust for runtime as rust then a strategy is not detected, It should detect the serverless function as func.yaml is present with warning message `“Support for <RUNTIME> is not yet available.”`
![image](https://user-images.githubusercontent.com/2561818/212012333-fd4f7610-6119-4e1a-95aa-3b70256ea908.png)